lighthouse/classes/event_properties/definitions.py

```diff
@@ -0,0 +1,213 @@
+from functools import cached_property
+from lighthouse.classes.messages.warehouse_messages import (  # type: ignore
+    ROLE_TYPE_CP_SOURCE_LABWARE,
+    SUBJECT_TYPE_PLATE,
+    ROLE_TYPE_ROBOT,
+    SUBJECT_TYPE_ROBOT,
+)
+from lighthouse.classes.event_properties.interfaces import EventPropertyAbstract, RetrievalError  # type: ignore
+from lighthouse.classes.event_properties.validations import SimpleEventPropertyMixin  # type: ignore
+from typing import Any, List, Dict
+from lighthouse.classes.services.cherrytrack import ServiceCherrytrackMixin  # type: ignore
+from lighthouse.classes.services.mongo import ServiceMongoMixin  # type: ignore
+from lighthouse.constants.fields import (
+    FIELD_CHERRYTRACK_LH_SAMPLE_UUID,
+    FIELD_EVENT_RUN_ID,
+    FIELD_EVENT_ROBOT,
+    FIELD_EVENT_USER_ID,
+    FIELD_EVENT_BARCODE,
+)
+
+from flask import current_app as app
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class RunID(EventPropertyAbstract, SimpleEventPropertyMixin):
+    def validate(self):
+        self.validate_param_not_missing(FIELD_EVENT_RUN_ID)
+        self.validate_param_is_integer(FIELD_EVENT_RUN_ID)
+        return self._validate
+
+    @cached_property
+    def value(self):
+        with self.retrieval_scope():
+            return self._params.get(FIELD_EVENT_RUN_ID)
+
+    def add_to_warehouse_message(self, message):
+        return None
```

Comment:
Could you use `pass` instead? Might make things clearer that this method does nothing and does not need to return anything?

Reply:
Yes, more correct

Reply:
Done